src/logistration/AccountActivationMessage.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { Alert } from '@edx/paragon';
+import PropTypes from 'prop-types';
+
+import { ACCOUNT_ACTIVATION_MESSAGE } from './data/constants';
+
+const AccountActivationMessage = (props) => {
+  const { messageType } = props;
+
+  const supportLink = (
+    <Alert.Link href="https://support.edx.org/hc/en-us/articles/227340127-Why-haven-t-I-received-my-activation-email-">
```

Comment:
@waheedahmed I almost forgot. What is the best way to add these urls? I don't want it hardcoded

Reply:
Thorugh an environment variable using remote configs.

Reply:
And yes, we should not hardcode edX specific things here.

Reply:
I have used env variable for this. Will create PR for edx-internal